src/main/java/pl/task/parkingmeter/repository/ValueRatesServiceImpl.java

```diff
@@ -0,0 +1,20 @@
+package pl.task.parkingmeter.repository;
+
+import org.springframework.stereotype.Service;
+import pl.task.parkingmeter.entity.Rates;
+import pl.task.parkingmeter.entity.ValueRate;
+
+@Service
+public class ValueRatesServiceImpl implements ValueRateService {
+
+    private ValueRateRepository repository;
+
+    public ValueRatesServiceImpl(ValueRateRepository repository) {
+        this.repository = repository;
+    }
+
+    @Override
+    public ValueRate findValueRateByHoursAndRatesId(long hours, Rates rates) {
+        return repository.findValueRateByHoursAndRates(hours, rates);
```

Comment:
czy jest sens robienia serwisu który jest 1:1 przelotką do db?

Reply:
daje mi to możliwość mockowania i zwiększa testowalność

Reply:
a repozytorium nie da się mockować?